src/Model/Config/PluginConfig.php

```diff
@@ -159,6 +160,7 @@ class PluginConfig extends ActiveRecord
      */
     public static $groups = [
```

Comment:
`public static array $groups = [` must be used here

Reply:
Thanks! will add.
PS: The line is not new and has no change regarding this PR!

Reply:
In this case, we have to also add array type in line 154 of this file!